app/Http/Livewire/Airfields/Index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Livewire\Airfields;
+
+use App\Http\Resources\UKCP\AirfieldCollection;
+use Illuminate\Support\Facades\Http;
+use Livewire\Component;
+
+class Index extends Component
+{
+    public string $search = '';
+
+    protected $queryString = ['search' => ['except' => ''], 'showEmpty' => ['except' => false]];
+
+    public bool $showEmpty = false;
+
+    public function render()
+    {
+        $url = $this->showEmpty ? '/admin/airfields?all=true' : '/admin/airfields';
+        $airfields = (new AirfieldCollection(Http::ukcp($url)['airfields']))->sortBy('code');
+
+        // allow to search based upon a match of the
+        $airfields_filtered = $this->search
+            ? $airfields->search($this->search)
+            : $airfields;
```

Comment:
If `$search` is `null`, the search method should handle that and just return everything rather than having to do a check on search

Reply:
done